src/site/content/en/lighthouse-best-practices/no-document-write/index.md

```diff
@@ -1,47 +1,61 @@
 ---
 layout: post
-title: Page uses document.write()
+title: Uses document.write()
 description: |
-  Learn about `no-document-write` audit.
+  Learn how to speed up your page's load time by avoiding  document.write().
 web_lighthouse:
   - no-document-write
+updated: 2019-08-28
 ---
 
 For users on slow connections,
 such as 2G, 3G, or slow Wi-Fi,
-external scripts dynamically injected via `document.write()`
+third-party scripts dynamically injected via `document.write()`
 can delay the display of main page content by tens of seconds.
-Lighthouse lists out every call to `document.write()`:
+Chrome therefore blocks the execution of these scripts.
+
+## How the Lighthouse `document.write()` audit fails
+
+[Lighthouse](https://developers.google.com/web/tools/lighthouse/) flags
+all calls to `document.write()`:
 
 <figure class="w-figure">
-  <img class="w-screenshot w-screenshot--filled" src="no-document-write.png" alt="Lighthouse audit showing usage of document.write">
-  <figcaption class="w-figcaption">
-    Page uses <code>document.write()</code>.
-  </figcaption>
+  <img class="w-screenshot" src="no-document-write.png" alt="Lighthouse audit showing usage of document.write">
 </figure>
 
-See [Intervening against `document.write()`](https://developers.google.com/web/updates/2016/08/removing-document-write)
-to learn more.
+Lighthouse reports calls to `document.write()` because it's often used to inject
+third-party render-blocking scripts. Chrome won't execute such scripts
```

Comment:
so mostly i'd say

1. lighthouse reports on doc.write because its super bad 
1. there's also an intervention which means you can't rely on this API reliably. go read about it. stop using the api, dummy. 
1. (the avoid section lgtm :)